tests/test_audio.py

```diff
@@ -227,11 +227,12 @@ async def test_send_all_args(self, bot, chat_id, audio_file, thumb_file):
         assert message.audio.thumb.height == Space.thumb_height
         assert message.has_protected_content
 
-    async def test_get_and_download(self, bot, audio):
+    async def test_get_and_download(self, bot, chat_id, audio):
         path = Path("telegram.mp3")
         if path.is_file():
             path.unlink()
 
+        audio = (await bot.send_audio(chat_id, audio.file_id, read_timeout=50)).audio
```

Comment:
why not add the read_timeout directly in the fixture?

Reply:
it is there in the fixture already

Reply:
then I don't quite unnderstand what these added lines do …

Reply:
the reason I added them is essentially the commit message of https://github.com/python-telegram-bot/python-telegram-bot/pull/3426/commits/4b55ee567ae31bef0f7a0a200daf50351829c8bf.

It's also due to this behaviour from Telegram I don't understand, see this code snippet:
``` python
# audio is a tg.Audio, now a "module" scope, hence same audio object used for all tests
audio1 = (await bot.send_audio(chat_id, audio.file_id, read_timeout=50)).audio
audio2 = (await bot.send_audio(chat_id, audio.file_id, read_timeout=50)).audio
new_file = await bot.get_file(audio.file_id)
print(audio.file_id == new_file.file_id)  # False  -- This should be true?!
print(audio1.file_id == new_file.file_id)  # False
print(audio2.file_id == new_file.file_id)  # True  -- Why is this True but the previous one is False?
print(audio1.file_id == audio2.file_id)  # False
```
since other tests send audio using that file_id, it seems to change file_id returned by `get_file`.

Reply:
So the problem is that files have multiple file ids (which is documented) and when exactly they change is not obvious and more importantly not documented. I would therefore suggest to remove tests that assume that the `file_id` stays unchanged. i.e. things like `assert (await bot.get_file(file_id)).file_id == file_id`. Even with your updated logic, TG could change how things behave und we run into problems again.